ballerina/tests/test.bal

```diff
@@ -1102,7 +1102,7 @@ function testVendorBillGetOperation() {
         recordType: "vendorBill"
     };
     VendorBill|error output = netsuiteClient->getVendorBillRecord(recordInfo);
```

Comment:
Do we need to check for this error case ? We can just update the method as following ?
```ballerina
function testVendorBillGetOperation() returns error? {
    // other logic
    VendorBill output = check netsuiteClient->getVendorBillRecord(recordInfo);
}
```

Reply:
https://github.com/ballerina-platform/module-ballerinax-netsuite/pull/93#discussion_r1325678188

Reply:
Agreed :+1: